cfpb/cfgov-refresh
# Filterable List: Allow Post date description field to be blank

## Changes

- Give the "Post date description" field a more descriptive label and help text
- Remove the default value and make it non-required, so it may be left blank.

## Testing

1. Pull down this branch, get a recent database dump, run all migrations
2. Visit a filterable list page, such as http://localhost:8000/policy-compliance/enforcement/actions/
    - notice that each post in the filterable list says "Date filed" before the date of the post
3. Open the page in the Wagtail admin: http://localhost:8000/admin/pages/1327/edit/
4. Delete all text ("date filed") in the "Date stamp descriptor" field
5. Publish the page
6. See that each post has no date description

## Screenshots

<img width="570" alt="Screen Shot 2019-09-27 at 10 06 44 AM" src="https://user-images.githubusercontent.com/4295388/65775739-c0167600-e10e-11e9-915d-16f7a4a12dc5.png">
<img width="735" alt="Screen Shot 2019-09-27 at 10 20 44 AM" src="https://user-images.githubusercontent.com/4295388/65776604-7d559d80-e110-11e9-845b-066ecdb0f1e8.png">


## Checklist
